cmd/cimp/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"path/filepath"
+
+	"github.com/humans-group/consul-importer/lib/cimp"
+)
+
+func main() {
+	pathRaw := flag.String("p", "./config-local.yaml", "Path to yaml with config-file which should be imported")
+	formatRaw := flag.String("f", "", "File format: json, yaml. If empty - got from extension. Default: yaml")
+	consulEndpoint := flag.String("c", "consul:8500", "Consul endpoint in format: `address:port`")
+	flag.Parse()
+	if pathRaw == nil || formatRaw == nil || consulEndpoint == nil {
+		panic("Impossible! Flags with defaults can't be nil")
+	}
+
+	path, err := filepath.Abs(*pathRaw)
+	check(err)
+
+	format, err := cimp.InitFormat(*formatRaw, path)
+	check(err)
+
+	kv := cimp.NewKV()
+	check(kv.FillFromFile(path, format))
+
+	serverName, err := kv.GetString("server/name")
```

Comment:
must be in the config, why general lib bounded to our internal setup?

Reply:
> must be in the config, why general lib bounded to our internal setup?

added as cli-param